lib/verdict/assignment.rb

```diff
@@ -1,12 +1,11 @@
 class Verdict::Assignment
-
   attr_reader :experiment, :subject_identifier, :group, :created_at
 
   def initialize(experiment, subject_identifier, group, originally_created_at, temporary = false)
     @experiment         = experiment
     @subject_identifier = subject_identifier
     @group              = group
-    @returning          = !originally_created_at.nil?
+    @first              = originally_created_at.nil? || experiment.timestamps_out_of_band?
```

Comment:
Yeah, the reason I flipped `@returning` to `@first` is because I didn't like `!originally_created_at.nil?` 

`ActiveSupport` is really great.